crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -945,7 +946,21 @@ impl EntityCommands<'_> {
     /// ```
     #[track_caller]
     pub fn insert(&mut self, bundle: impl Bundle) -> &mut Self {
-        self.add(insert(bundle))
+        self.add(insert(bundle, InsertMode::Replace))
+    }
+
+    /// Adds a [`Bundle`] of components to the entity without overwriting.
+    ///
+    /// This is the same as [`insert`], but in case of duplicate components
```

Comment:
```suggestion
    /// This is the same as [`Commands::insert`], but in case of duplicate components
```

`insert` will instead refer to the module-level private function called `insert`, not the method on `Commands`